rust/aiconfigurator-core/src/models/deepseek.rs

```
@@ -0,0 +1,450 @@
+    let layers = cfg.spec.num_hidden_layers as f64;
+    let h = cfg.spec.hidden_size;
+    let tp = cfg.parallel.tp_size.max(1);
+    let vocab_per_tp = cfg.spec.vocab_size / tp;
+    let dtypes = cfg.dtypes;
+    let num_experts = cfg.spec.num_experts.max(1);
+    let topk = cfg.spec.top_k.max(1);
+    let moe_inter = cfg.spec.moe_intermediate_size;
+    let moe_tp = cfg.parallel.moe_tp_size.max(1);
+    let moe_ep = cfg.parallel.moe_ep_size.max(1);
+    let attn_dp = cfg.parallel.attention_dp_size.max(1);
+    let n_per_tp = cfg.spec.num_attention_heads / tp;
```

Clippy (1.95.0)

[warning] unused variable

(warning)